app/views/pets/show.html.erb

```diff
@@ -9,6 +8,13 @@
       <%= flash[:notice] %>
     <% end %>
   </div>
+
+  <div>
+    <% if @pet.status == "pending" %>
+      <p>This pet is currently on hold for: <%= @applicants.map {|applicant| applicant.name}.pop %></p>
```

Comment:
Not sure how we know that the last item in the array (returned by `pop`) is for sure the person in question for this pet. I'm going to merge anyway and then take a look.